postcode_lookup/templates/includes/postal_votes.html

```diff
@@ -0,0 +1,53 @@
+<{{ section.context.htag_primary }} id="{{toc_id}}">{% trans %}Voting by post{% endtrans %}</{{ section.context.htag_primary }}>
+{% if section.context.can_register %}
+    <{{ section.context.htag_secondary }}>{% trans %}Apply for a postal vote{% endtrans %}</{{ section.context.htag_secondary }}>
+    <p>
+      {% trans %}
+        If you know that you won't be able to get to the polling station on polling day, you can vote by post.
+      {% endtrans %}
+      {% trans date=section.timetable.postal_vote_application_deadline|date_filter %}
+        You have until {{ date }}.
+      {% endtrans %}
+    </p>
+    <p>
+      <a href="https://www.gov.uk/apply-postal-vote" class="o-button o-button--primary o-button--dark o-external-link">
+        {% trans %}Apply for a postal vote{% endtrans %}
+      </a>
+    </p>
+    <p>
+      {% trans %}
+        <a href="https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/apply-vote-post">Read more</a> about voting by post.
+      {% endtrans %}
+    </p>
+{% else %}
+    {%
+        trans
+        date=section.data.date|date_filter,
+        application_deadline=section.timetable.postal_vote_application_deadline|date_filter
+    %}
+    <p>
+      The deadline to apply for a postal vote for the elections taking place on {{ date }} was {{ application_deadline }}.
+      You can <a href="https://www.gov.uk/apply-postal-vote">apply for a postal vote</a> in future elections.
+    </p>
+    {% endtrans %}
+{% endif %}
+<{{ section.context.htag_secondary }}>{% trans %}If you have a postal vote{% endtrans %}</{{ section.context.htag_secondary }}>
+<p>
+  {% if section.context.dispatch_dates %}
+    {%
+      trans
+      date0=section.context.dispatch_dates.0|date_filter,
+      date1=section.context.dispatch_dates.1|date_filter,
+      date2=section.context.dispatch_dates.2|date_filter
+    %}
+      If you applied for your postal vote by {{ date0 }}, your local council will send your postal voting pack from {{ date1 }}.
```

Comment:
I still feel like the messaging here is a bit unclear if you applied for your postal vote after this date but before the deadline
See comments in https://democracyclub.slack.com/archives/C93TB8466/p1743599590093529